### ISSUE TITLE
[High Contrast]: Icon Button: style is different from expected.

### ISSUE THREAD

**Issue by bhaveshbhati** [OfficeDev/office-ui-fabric-react#1687]
### Bug Report

- __Package version(s)__: latest
- __Browser and OS versions__: edge, Windows 10

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (No)

Requested priority: (Normal) 

Products/sites affected: (if applicable)

#### Describe the issue:

If I observe the disabled Command button and Icon button, while the command button shows the icon in disabled text color but the icon button doesn't show it in disabled text color. 

![image](https://cloud.githubusercontent.com/assets/1016447/25650394/3666541a-2ffa-11e7-97da-203573f1739c.png)

#### Actual behavior:

![image](https://cloud.githubusercontent.com/assets/1016447/25650377/148b76ae-2ffa-11e7-9d12-ec2675d580b4.png)

#### Expected behavior:

Both should have been in disabled text color.
